### PR TITLE
ft(admin signup): admin sign-up controller

### PR DESCRIPTION
**Pull request by Mohamilr**
i wrote tests and the controller for caterer(admin signup). review and let me know what should be improved befor merging

image

### CODE REVIEW

File: src/models/db.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import pg from 'pg';
+
+const connection = {
```
Comment (Rexben001) [inline src/models/db.js:6]: How do you intend to connect it with your production database?
Simply create a .env file, add your db connection strings there and use `dotenv` to access them

**Review by Rexben001 — CHANGES_REQUESTED** [review]

**Comment by Rexben001** [comment]
@Mohamilr Good job 🔥 
Endeavour to commit often

File: src/models/db.js (hunk 1 above)
Comment (Mohamilr) [reply src/models/db.js:6]: I actually did, but forgot to remove the direct values. If you noticed, I commented out the environment connection.

Comment (Mohamilr) [reply src/models/db.js:6]: I will do that in the next commit. Can the pull request before merged now?

Comment (Rexben001) [reply src/models/db.js:6]: Make changes and push it. Then, I'll merge it

File: package.json [hunk 2]
```diff
@@ -4,8 +4,9 @@
   "description": "Book-A-Meal is an application that allows customers to make food orders and helps the food",
   "main": "server.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "start-dev": "nodemon --exec babel-node src/server.js"
+    "start-dev": "nodemon --exec babel-node src/server.js",
```
Comment (Rexben001) [inline package.json:8]: I can't see a start script ??

Comment (Mohamilr) [reply package.json:8]: thats is what the start-dev does

Comment (Rexben001) [reply package.json:8]: Heroku looks for `start` script and not `start-dev`. You can't used `nodemon` into production @Mohamilr

Comment (Mohamilr) [reply package.json:8]: done sir

**Review by Rexben001 — APPROVED** [review]

**Pull request merged**